site/content/docs/4.3/utilities/vertical-align.md

```diff
@@ -9,6 +9,8 @@ Change the alignment of elements with the [`vertical-alignment`](https://develop
 
 Choose from `.align-baseline`, `.align-top`, `.align-middle`, `.align-bottom`, `.align-text-bottom`, and `.align-text-top` as needed.
 
+If you are looking for flex-alignment, checkout: [flex-alignment]({{< docsref "/utilities/flex#auto-margins" >}}).
```

Comment:
`checkout` doesn't seem right AFAICT, but I'm not a native English speaker either. I'd go with `have a look at ...` or maybe just `check ...`, both without the comma

Reply:
I believe it would be "check out" instead of "checkout".